### ISSUE TITLE
Possibly reduntant Update call in uFrameComponent

### ISSUE THREAD

**Issue by mdsolver** [micahosborne/uFrame#3]
`uFrameComponent` declares an empty private non-virtual Update(), which is wired into Unity update loop per each component in the scene and called each frame. I have verified this behavior in editor play mode.

Why is this call in there? Should it be removed?

**Comment by micahosborne** [comment]
Yeah thats definitely not needed anymore. Must of been overlooked.  Might be a good idea to remove it if its empty.
